tools/api-linter/tests/default-config-expected-output.txt

```diff
@@ -1,6 +1,3 @@
-This build understands rustdoc format version 10
-Running rustdoc to produce json doc output...
-Examining all public types...
```

Comment:
does rustdoc include an output format version number? would be good to assert on that probably